src/pages/Build.vue

```diff
@@ -848,6 +876,9 @@ export default defineComponent({
         return false
       }
       return true
+    },
+    moduleEVR(module) {
```

Comment:
it should be NVSCA, name-version-context-arch

Reply:
Done